hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DiskBalancerStatus.java

```diff
@@ -45,4 +45,4 @@ public boolean isRunning() {
   public DiskBalancerConfiguration getDiskBalancerConfiguration() {
     return diskBalancerConfiguration;
   }
-}
\ No newline at end of file
+}
```

Comment:
Please revert this unnecessary change in this and the following two files.

Reply:
The checkstyle will prompt the following warnings:
```
hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DiskBalancerStatus.java
 1: File does not end with a newline.
```